### PR TITLE
In progress sizing tweaks for wizard - see #2207

### PR DESCRIPTION
**Pull request by timlinux**
Still a work in progress - not ready to merge

### CODE REVIEW

**Comment by timlinux** [comment]
screen shot 2015-08-25 at 12 00 08

I have fixed the resize issue and added styling updates to match the new styling for the web site etc.

File: safe/gui/tools/wizard_dialog.py [hunk 1]
```diff
@@ -663,7 +423,8 @@ def resizeEvent(self, ev):
         .. note:: This is an automatic Qt slot
            executed when the window size changes.
         """
-        self.update_MessageViewer_size()
+        pass
+        # self.update_MessageViewer_size()
```
Comment (cchristelis) [inline safe/gui/tools/wizard_dialog.py]: this line looks unneeded

File: safe/gui/tools/wizard_strings.py [hunk 2]
```diff
@@ -0,0 +1,173 @@
+from safe.utilities.i18n import tr
```
Comment (cchristelis) [inline safe/gui/tools/wizard_strings.py]: perhaps we could add the inasafe header.

File: safe/gui/tools/wizard_dialog.py [hunk 3]
```diff
@@ -3936,7 +3698,7 @@ def on_pbnNext_released(self):
 
         # Run analysis after switching to the new step
         if new_step == step_fc_analysis:
-            self.update_MessageViewer_size()
+            #self.update_MessageViewer_size()
```
Comment (cchristelis) [inline safe/gui/tools/wizard_dialog.py]: this line looks unneeded.

**Comment by cchristelis** [comment]
LGTM

**Comment by timlinux** [comment]
OK thanks @cchristelis - because I want to get all the main changes done ASAP, I will address your comments in a separate PR

**Pull request merged**